docs/src/Status.doc.js

```diff
@@ -0,0 +1,86 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Status } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
```

Comment:
Don't forget to add a card up here for FeedbackCallout

Reply:
Done!